weave

```diff
@@ -1066,10 +1081,7 @@ http_call_unix() {
 
 call_weave() {
     check_running $CONTAINER_NAME || return 1
-    if ! http_call $HTTP_ADDR "$@" ; then
-        echo "Call to $CONTAINER_NAME failed." >&2
-        return 1
-    fi
+    http_call $HTTP_ADDR "$@"
```

Comment:
The ""Call to $CONTAINER_NAME failed." error message here provided some (though arguably not enough) context. Without it, we can get weave commands to fail with errrors like
```
curl: (7) Couldn't connect to server
```
which isn't particularly illuminating.

OTOH, I guess we don't want to show an error message in some (all?) 404 cases. Hmm.